### PR TITLE
feat: add Lambda Ephemeral storage to LambdaWorker props

### PR DESCRIPTION
**Pull request by ShaunParsons**
Add support for `ephemeralStorageSize` in the `LambdaWorker` construct.

### CODE REVIEW

File: lib/lambda-worker/lambda-worker-props.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export interface LambdaWorkerProps {
     entry?: string;
     enableQueue?: boolean;
     environment?: { [key: string]: string };
+    ephemeralStorageSize?: cdk.Size;
```
Comment (stephenrob) [inline lib/lambda-worker/lambda-worker-props.ts:23]: Do we need to add any validation that it is less than 10240 or does the underlying L2 construct handle this

**Review by stephenrob — APPROVED** [review]
Looks good - just a non blocking question on validation of the prop

File: lib/lambda-worker/lambda-worker-props.ts (hunk 1 above)
Comment (ShaunParsons) [reply lib/lambda-worker/lambda-worker-props.ts:23]: Yup: https://github.com/aws/aws-cdk/blob/1ef0af8dfe6dfb9653338ea9daa6a60d8cbd8ccb/packages/%40aws-cdk/aws-lambda/lib/function.ts#L766-L769

**Review by malcyL — APPROVED** [review]

**Pull request merged**